port_ocean/run.py

```diff
@@ -49,9 +49,8 @@ def run(
     if initialize_port_resources is not None:
         app.config.initialize_port_resources = initialize_port_resources
 
-    if app.config.initialize_port_resources:
```

Comment:
Why did u remove feature flags?

Reply:
because then it wont enter the method and skipp the whole initializtion
i moved this if into the function